Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -33,7 +33,7 @@ public struct WorldScaleSceneView: View {
     /// The camera controller that will be set on the scene view.
     @State private var cameraController: TransformationMatrixCameraController
     /// The view model for the calibration view.
-    @StateObject private var calibrationViewModel = WorldScaleCalibrationViewModel()
+    @State private var calibrationViewModel: WorldScaleCalibrationViewModel
```

Comment:
Sorry for the delayed response. The thing that held me back a few days ago was the decision on whether this should be a state or a state object. Since this type is a class, shall we use StateObject [initialization](https://developer.apple.com/documentation/swiftui/stateobject#Initialize-state-objects-using-external-data)?

Reply:
This suggestions works, although the variable needs to be declared as a `@StateObject` in order to use StateObject [initialization](https://developer.apple.com/documentation/swiftui/stateobject#Initialize-state-objects-using-external-data). I can make this change.

Reply:
I don't think it needs to be a `StateObject` here because it's not being observed here. It's being observed in other views. That's why I changed it to `State`. Also recall other classes (ie `Map`) are `@State`.

Reply:
I agree now. We aren't using StateObject's observation capability in `WorldScaleSceneView`, so StateObject has no benefit over State in this particular case.